src/content/1.7/modules/sell/techvalidation-keysteps.md

```diff
@@ -34,7 +34,10 @@ After meeting each requirement provided by the Validator, here's a list of extra
 * The presence of DROP/ALTER of PrestaShop tables.
     * It's highly forbidden to apply any changes on PrestaShop Core tables. It's very very dangerous and we don't allow any risk.
 
-* Except for [Payment Modules](https://github.com/PrestaShop/paymentexample/blob/master/paymentexample.php#L150), the use of iframes is STRICTLY FORBIDDEN.
+* The use of iframes is highly discouraged for security reasons, although they are implemented in different part of the core like in [Payment Modules](https://github.com/PrestaShop/paymentexample/blob/master/paymentexample.php#L150).
+    * Using an iframe authorizes to load content from a site that is not controlled by us. This is the same problem as authorizing to load javascript file from an external source. If you're being hacked, the attacker could potentially exploit other failures to take control of all the shops that would have installed the module.
```

Comment:
```suggestion
    * Using an iframe authorizes to load content from a site that is not controlled by the PrestaShop app. This is the same problem as authorizing to load a javascript file from an external source. If the file is being hacked, the attacker could potentially exploit other failures to take control of your shop and all the shops that would have installed the module.
```

Reply:
What about `If the source is being hacked` ?

Reply:
As we talk to a module developer and not a merchant, I unfortunately disagree with the change `your shop`

Reply:
GTM :+1:

Reply:
Updated